deepinv/benchmark/urban100_gaussian_denoising.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+import torch
+import torch.utils.data
+import deepinv as dinv
+import torchvision.transforms as transforms
+import numpy as np
+from tqdm.auto import tqdm
+from typing import Any
+import pandas as pd  # noqa: TID253
+
+
+class Benchmark:
+    r"""
+    Benchmark for Gaussian Denoising on Urban100 dataset
+
+    .. note::
+
+        The noise standard deviation is set to 25/255 for images normalized between 0 and 1.
+    """
+
+    @staticmethod
+    def run(
+        model: dinv.models.Denoiser,
+        *,
+        device: torch.device | str = torch.device("cpu"),
+    ) -> Any:
+        """Run the benchmark on the given model"""
+        dataset = dinv.datasets.Urban100HR(
+            "data/Urban100", download=True, transform=transforms.ToTensor()
+        )
+
+        rng = torch.Generator(device)
+        physics = dinv.physics.Denoising(
+            dinv.physics.GaussianNoise(sigma=25 / 255, rng=rng)
+        ).to(device)
+
+        psnr_fn = dinv.metric.PSNR(min_pixel=0.0, max_pixel=1.0).to(device)
```

Comment:
would be nice to use modality-specific metrics e.g. our ERGAS etc. for spectral imaging benchmarks